repository/c.json

```diff
@@ -1604,6 +1604,18 @@
 				}
 			]
 		},
+		{
+			"name": "CloseAllNoConfirm",
+			"details": "https://github.com/ddbln/CloseAllNoConfirm",
+			"description": "Instantly close all files (saved & unsaved) without confirmations.",
```

Comment:
You don't need to provide a description if it's the same as your GitHub repo's description. That way you can easily change it. In fact, even the name can be omitted in your case.

Reply:
Thank you!